api/user/comments/model.py

```diff
@@ -1,21 +1,32 @@
 from dataclasses import asdict, dataclass
 
 from api.app import db
-from api.utils import generate_object_ID
+from api.utils import generate_object_ID, get_author_info
 
 
 @dataclass
 class Comment(db.Model):
     id: int = db.Column(db.String(50), primary_key=True, default="")
-    created: str = db.Column("created", db.String(20), nullable=False)
+    published: str = db.Column("published", db.String(20), nullable=False)
     contentType: str = db.Column("contentType", db.String(50), nullable=False)
-    content: str = db.Column("content", db.String(50), nullable=False)
+    comment: str = db.Column("comment", db.String(50), nullable=False)
     # Foreign Key
-    author_id: int = db.Column("author_id", db.Integer, nullable=False)
-    post_url: int = db.Column("post_url", db.Integer, db.ForeignKey("post.url"), nullable=False)
+    author_id: int = db.Column("author_id", db.String(50), nullable=False)
+    post_url: int = db.Column("post_url", db.String(50), db.ForeignKey("post.url"), nullable=False)
```

Comment:
can we derive this? I'm always a fan of deriving rather than storing everything (for example, lets say we want to use an existing database for a new server instance for testing: the URL will be different and every row in the db would need to be update to reflect that)

Reply:
I changed it to post id now